src/search.cpp

```diff
@@ -826,6 +826,7 @@ namespace {
       }
       else
           ++moveCount;
+          ss->moveCount = moveCount;
```

Comment:
The indentation and the brackets aren't matching up here.  Is this intended to only execute in the else branch?